main.mk

```diff
@@ -66,7 +66,7 @@ $(LIBDIR)/clean-for-DTD.xslt: $(LIBDIR)/rfc2629xslt/clean-for-DTD.xslt
 
 $(XSLTDIR)/clean-for-DTD.xslt $(XSLTDIR)/rfc2629.xslt: $(XSLTDIR)
 $(XSLTDIR):
-	git clone --depth 10 -b master https://github.com/reschke/xml2rfc $@
+	git clone --depth 10 $(XSLT_CLONE_ARGS) -b master https://github.com/reschke/xml2rfc $@
```

Comment:
You need to pass this variable via the circle config

Reply:
Whoops, I thought I had.  I'll clean that up when I look at the below.

Reply:
I see what I did wrong.  I made my updates to the config.yml in the http-extension repo, rather than in the template.